packages/clerk-js/src/core/resources/OrganizationSettings.ts

```diff
@@ -11,10 +11,9 @@ export class OrganizationSettings extends BaseResource implements OrganizationSe
     this.fromJSON(data);
   }
 
-  protected fromJSON(data: OrganizationSettingsJSON): this {
-    data = data || {};
-    this.enabled = data.enabled || false;
-    this.maxAllowedMemberships = data.max_allowed_memberships || 0;
+  protected fromJSON(data: OrganizationSettingsJSON | null): this {
```

Comment:
This looks to be handled differently than the rest. Since it looks like we need some default values, we should probably make it more explicit:
```tsx
data = data || { enabled: false,  maxAllowedMemberships: 0 } 
this.enabled = data.enabled;
this.maxAllowedMemberships = data.max_allowed_memberships;
```